src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -666,6 +691,12 @@
 
         #region Expression Visitors
 
+        public override BoundNode VisitLoweredIsPatternExpression(BoundLoweredIsPatternExpression node)
+        {
```

Comment:
>// the spilling will occur in the enclosing node. [](http://example.com/codeflow?start=12&length=49)

This doesn't feel right to me. See my previous comment about triggering unconditional spilling for this node.